companies/slovnaftbajk.json

```diff
@@ -0,0 +1,19 @@
+{
+    "slug": "slovnaftbajk",
+    "relevant-countries": [
+        "sk"
+    ],
+    "categories": [
+        "travel"
+    ],
+    "name": "Slovnaft Mobility Services, s. r. o.",
+    "address": "Vlčie hrdlo 1\n82412 Bratislava\nSlovakia",
+    "phone": "+421 2 5950 5950",
```

Comment:
I didn't find the phone number in the privacy policy. Please add https://slovnaftbajk.sk/en/contact-us.html to the sources.

Reply:
source added